coralproject/talk
# [CORL-1358] bios

## What does this PR do?

Adds support for commenter bios, which must be enabled through config. 



## What changes to the GraphQL/Database Schema does this PR introduce?

Adds new `settings.memberBios` config option on tenant, and `bio` field on users. 



## How do I test this PR?

- enable bios from Configuration -> General
- add a bio from stream by going to the preferences tab
- observe bio is visible when clicking on username in stream, in user drawer in admin.
- observe new "Bio is offensive or abusive" option for reporting a comment
